core/templates/dev/head/pages/exploration_player/StatsReportingService.js

```diff
@@ -53,9 +77,24 @@ oppia.factory('StatsReportingService', [
         });
     };
 
+    if (ENABLE_NEW_STATS_FRAMEWORK && !_editorPreviewMode && (
+        !explorationIsComplete)) {
```

Comment:
Why would the exploration be complete when this service is initialized? Is the aim here to deal with 1-state explorations?

(Not saying it's wrong, necessarily -- just that I'm not sure I understand the intent.)

Reply:
Hmm.. Yeah, I don't think we need this.